### PR TITLE
feat: add chart version input on bug report to help investigation

### PR DESCRIPTION
**Pull request by yu-croco**
On issues, the report of `helm chart version`  will help us to investigate more efficiently.
So I added the input form for them. 🙋 

---

Signed-off-by: yu-croco <yuki.kita22@gmail.com>

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/main/community#contributing-to-argo).
* [x] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

Changes are automatically published when merged to `main`. They are not published on branches.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -26,6 +26,22 @@ body:
   validations:
     required: true
 
+- type: input
+  attributes:
+    label: Helm chart version
+    description: Helm chart version that you encounterd problem.
+    placeholder: ex. 0.16.2
```
Comment (mkilchhofer) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:33]:
```suggestion
    placeholder: E.g. 0.16.2
```

I am not an english expert but I think "ex" is an abbreviation for exercise.

Comment (yu-croco) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: Thank you for reviewing!
I fixed on https://github.com/argoproj/argo-helm/pull/1318/commits/3aea2fc6e15ca7036c46cbdd43e76eeba894c357 (exiting formats were all lowercase, so I followed them) . 🙋

**Comment by mkilchhofer** [comment]
Maybe we can omit the appVersion as we normally want that the user uses the bundled version inside the charts appVersion?

**Comment by yu-croco** [comment]
We can override by setting `xxxx.image.tag` but I think it's a rare case.😇  You are right. 
I removed it on https://github.com/argoproj/argo-helm/pull/1318/commits/4872adff14e640e918e68d53717fa5c65ebba3cf .

**Review by mbevc1 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 2]
```diff
@@ -26,6 +26,14 @@ body:
   validations:
     required: true
 
+- type: input
+  attributes:
+    label: Helm chart version
+    description: Helm chart version that you encounterd problem.
```
Comment (mkilchhofer) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:32]: Typo encounterd -> encountered. Maybe we better use a sentence like this:

```suggestion
    description: Version of the Helm chart this issue relates to
```

**Review by mkilchhofer — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 2 above)
Comment (yu-croco) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: I fixed on https://github.com/argoproj/argo-helm/pull/1318/commits/160691bf74aeba807cf7a503726f531a5a99a98b 🙇🙇

**Review by mkilchhofer — APPROVED** [review]

**Pull request merged**